addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiSystemProperties.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.kodi.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Class representing Kodi system properties (https://kodi.wiki/view/JSON-RPC_API/v9#System.Property.Value)
+ *
+ * @author Christoph Weitkamp - Initial contribution
+ */
+@NonNullByDefault
+public class KodiSystemProperties {
+    private boolean canhibernate;
+    private boolean canreboot;
+    private boolean cansuspend;
+    private boolean canshutdown;
+
+    public boolean canHibernate() {
+        return canhibernate;
+    }
+
+    public void setCanhibernate(boolean canhibernate) {
+        this.canhibernate = canhibernate;
+    }
+
+    public boolean canReboot() {
+        return canreboot;
+    }
+
+    public void setCanreboot(boolean canreboot) {
+        this.canreboot = canreboot;
+    }
+
+    public boolean canSuspend() {
+        return cansuspend;
+    }
+
+    public void setCansuspend(boolean cansuspend) {
```

Comment:
cansuspend or canSuspend

Reply:
Should we use a GSON `@serialized` annotation

Reply:
@martinvw Are you talking about the [SerializedName](https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html) annotation? If not I am afraid I do not know what you like to tell me.

Reply:
Yes, that one :-)

Reply:
I have to if I want to be compliant with our coding guidelines :wink: .